cmd/web.go

```diff
@@ -309,7 +309,7 @@ func runWeb(ctx *cli.Context) error {
 				return
 			}
 		})
-		m.Post("/issues/attachments", repo.UploadIssueAttachment)
+		m.Post("/attachments", repo.UploadIssueAttachment)
```

Comment:
Agreed, will change the function name.  The function `repo.UploadIssueAttachment` does nothing to do with Issues.  

For future reference I changed this because I saw three options:

- Have issues uploading attachments to `/issues/attachments`
- add `/releases/attachments` and point to the same function or copy/paste function
- rationalise it and change the existing URL to just `/attachments`

Given that the GET url is already `/attachments` the third option seemed the most rational.

 a second URL fo